cpu/samd21/cmsis/samr21/include/cpu_conf.h

```diff
@@ -20,7 +20,7 @@
 
 #include "cpu_conf_common.h"
 
-#include "samd21.h"
+#include "samr21.h"
```

Comment:
Good point, but ...  what will the define be? We already have: CPU=samd21, CPU_MODEL=samd21j18a, CPU_ARCH=?, CPU_FAM=? and possibly more.